variables.tf

```
@@ -16,6 +16,17 @@ variable "environment_variables" {
   description = "A list of maps, that contain the keys 'name', 'value', and 'type' to be used as additional environment variables for the build. Valid types are 'PLAINTEXT', 'PARAMETER_STORE', or 'SECRETS_MANAGER'"
 }
 
+variable "description" {
+  default     = ""
```

```suggestion
  default     = "Managed by Terraform"
```
let's define some value